### PR TITLE
 layout_product-cards

### PR DESCRIPTION
**Pull request by SweeeD1**
- [DEMO LINK] https://SweeeD1.github.io/layout_product-cards/
- [TEST REPORT LINK] https://SweeeD1.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+      <img
+      src="images/imac.jpeg"
+      alt="imac"
+      class="card__imac card_imac-position"
+      >
+      <div class="product">
+        <p class="product__type product_type-position">
+          APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="product_code product__code-position">
```
Comment (volodymyr-soltys97) [inline src/index.html:24]: The class 'product_code' should be in BEM format like 'product__code'.

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+          Product code: 195434
+        </p>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <p class="stars__reviews stars_reviews-position">
```
Comment (volodymyr-soltys97) [inline src/index.html:34]: The class 'stars_reviews-position' should be in BEM format like 'stars__reviews-position'.

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+          Reviews: 5
+        </p>
+      </div>
+      <div class="price">
+        <p class = "price__text">
```
Comment (volodymyr-soltys97) [inline src/index.html:39]: Remove the space between 'class' and '='.

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,57 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="imac"
+      class="card__imac card_imac-position"
```
Comment (volodymyr-soltys97) [inline src/index.html:18]:
```suggestion
        src="images/imac.jpeg"
        alt="imac"
        class="card__imac card_imac-position"
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Your links are not working, also need to remove indentation here. Check it
image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,59 @@
...
+
+      <div class="price">
+        <p class="price__text">
+          Price:
+        </p>
+        <p class="price_number">
+          $2,199
+        </p>
+      </div>
+      <a
```
Comment (mykhalenych) [inline src/index.html:50]: Screenshot 2023-05-15 at 17 49 27

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  --secondary-color: #616070;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+
```
Comment (mykhalenych) [inline src/style.css:23]:
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+  padding: 32px 16px 16px;
+
```
Comment (mykhalenych) [inline src/style.css:25]:
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+  display: flex;
+
```
Comment (mykhalenych) [inline src/style.css:27]:
```suggestion
```

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product__code {
+  font-size: var(--font-size);
+  font-weight: 400;
+  color: var(--secondary-color);
+
```
Comment (mykhalenych) [inline src/style.css:51]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+  display: flex;
+  align-items: center;
+
+  margin-top: 23px;
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+  font-weight: 400;
```
Comment (mykhalenych) [inline src/style.css:66]: it is by default, delete it

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,61 @@
...
+        href="#buy"
+        class="buy-button"
+        data-qa="hover"
+      >
+          BUY
+      </a>
```
Comment (vlad-demchuk) [inline src/index.html:58]:
```suggestion
      >
        BUY
      </a>
```

**Review by vlad-demchuk — APPROVED** [review]
do not forget to create separate files for each bem block

**Pull request closed without merging**